lib/jxl/decode.cc

```diff
@@ -872,6 +874,15 @@ std::unique_ptr<BitReader, std::function<void(BitReader*)>> GetBitReader(
       });
 }
 
+bool CheckSizeLimit(JxlDecoder* dec, size_t xsize, size_t ysize) {
```

Comment:
Is it necessary for this function to be visible through linking, or can it be made static or anonymous namespace?

Reply:
Ooops! Fixed.